rados/conn.go

```diff
@@ -38,13 +38,13 @@ func (c *Conn) Cluster() ClusterRef {
 
 // PingMonitor sends a ping to a monitor and returns the reply.
 func (c *Conn) PingMonitor(id string) (string, error) {
-	c_id := C.CString(id)
-	defer C.free(unsafe.Pointer(c_id))
+	cid := C.CString(id)
```

Comment:
My inner Sheldon is asking me: why not `cId`? 😅

Reply:
I'll keep it in mind for next time.